edx_exams/apps/core/management/commands/bulk_add_course_staff.py

```diff
@@ -0,0 +1,98 @@
+"""Management command to populate the CourseStaffRole model and related User objects, from LMS, using CSV"""
+import logging
+import time
+
+import unicodecsv
+from django.core.management.base import BaseCommand
+from django.db import transaction
+
+from edx_exams.apps.core.models import CourseStaffRole, User
+
+logger = logging.getLogger(__name__)
+
+
+class Command(BaseCommand):
+    """
+    Management command to add Course Staff (and User) in batches from CSV
+    """
+    help = """
+    Add Course Staff in bulk from CSV.
+    Expects that the data will be provided in a csv file format with the first row
+    being the header and columns being: username, email, role, course_id.
+    Example:
+        $ ... bulk_add_course_staff --csv_path=foo.csv
+    """
+
+    def add_arguments(self, parser):
+        parser.add_argument(
```

Comment:
Should this argument be optional? I see that this management command is a no-op if you don't provide a CSV path,  but it suggests that something happened when it didn't (i.e. "Bulk add course staff complete!"). What do you think of either requiring the argument and making it a positional one or logging a different message when no CSV path is provided that indicates that it was a no-op?

Reply:
You're right, this should probably be required! I had referenced code from a couple of places and it must have been not required elsewhere. But for this case, it makes sense to require.